### ISSUE TITLE
Make proposal for CommonMark bidi support

### ISSUE THREAD

**Issue by ahangarha** [ahangarha/markdown-it-bidi#15]
We hope the success of this plugin can give confidence to CommonMark to add bidi support as their standard.

**Comment by majortom64** [comment]
Having BiDi support in CommonMark would be really useful.

**Comment by ahangarha** [comment]
> Having BiDi support in CommonMark would be really useful.

Yes. But recently they refused to do so. I request you share your thoughts on their forum.
But besides that, we need to make such a proposal; either they accept it or otherwise, it becomes the foundation for building extensions and plugins for the tools using solutions based on CommonMark
